dissect/target/plugins/os/windows/amcache.py

```diff
@@ -607,6 +620,28 @@ def device_containers(self):
         if self.amcache:
             yield from self.parse_inventory_device_container()
 
+    @export(record=AppLaunchAppcompatRecord)
+    def applaunches(self):
+        """Return AppLaunchAppcompatRecord records from Amcache applaunch files (Windows 11 22H2 or later).
+
+        TODO: Research C:\\Windows\\appcompat\\pca\\PcaGeneralDb0.txt and
```

Comment:
Will this be a TODO to be fixed within this MR or a future one?

Reply:
This would be a TODO for the future.